bitnami/wordpress/templates/cronjobs.yaml

```diff
@@ -0,0 +1,250 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.cronJobs.scheduler.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "common.names.fullname" . }}-scheduler
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+spec:
+  schedule: {{ .Values.cronJobs.scheduler.schedule | quote }}
+  concurrencyPolicy: {{ .Values.cronJobs.scheduler.concurrencyPolicy }}
+  failedJobsHistoryLimit: {{ .Values.cronJobs.scheduler.failedJobsHistoryLimit }}
+  successfulJobsHistoryLimit: {{ .Values.cronJobs.scheduler.successfulJobsHistoryLimit }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            {{- if .Values.cronJobs.scheduler.podLabels }}
+              {{- include "common.tplvalues.render" ( dict "value" .Values.cronJobs.scheduler.podLabels "context" $ ) | nindent 12 }}
+            {{- end }}
```

Comment:
Use "common.labels.standard" helper merging `.Values.commonLabels` with `.Values.cronJobs.scheduler.podLabels`. Also, consider annotations.
See how others charts do it:

- https://github.com/bitnami/charts/blob/main/bitnami/redmine/templates/cronjob.yaml#L24-L30